src/convert/transformers/staticResourceMetadataTransformer.ts

```diff
@@ -141,7 +141,15 @@ export class StaticResourceMetadataTransformer extends BaseMetadataTransformer {
   }
 
   private async getContentType(component: SourceComponent): Promise<string> {
-    return ((await component.parseXml()).StaticResource as JsonMap).contentType as string;
+    try {
+      return ((await component.parseXml()).StaticResource as JsonMap).contentType as string;
```

Comment:
It looks like you could create or pass in some type for this since the signature is
```
  public async parseXml<T = JsonMap>(): Promise<T> {
```

like component.parseXml<{StaticResource: {contentType: string;};}>()